openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_publish.json

```diff
@@ -313,6 +313,16 @@
                 }
             ]
         },
+        {
+            "type": "dict-modifiable",
+            "collapsible": true,
+            "key": "ValidatePathForPlugin",
+            "label": "Non-existent Paths in Non-Maya Nodes",
+            "use_label_wrap": true,
+            "object_type": {
+                "type": "text"
+            }
+        },
```

Comment:
I believe this is the wrong settings schema for this plugin. Instead it should be like the other plugins where it can also have `optional`, `enabled` , etc. for the `ValidatePathForPlugin` key.

As such, the `attributes` it should validate should be a subkey, like `project_settings/maya/publish/ValidatePathForPlugin/attributes`

@antirotor - thoughts?

Reply:
sure, something like:

```javascript
{
            "type": "dict",
            "collapsible": true,
            "key": "ValidatePathForPlugin",
            "label": "Non-existent Paths in Non-Maya Nodes",
            "children": [
            {
                "type": "boolean",
                "key": "enabled",
                "label": "Enabled"
            },
            {
                "type": "boolean",
                "key": "optional",
                "label": "Optional"
            },
            {
                "type": "boolean",
                "key": "active",
                "label": "Active"
            },
            {
                "type": "dict-modifiable",
                "store_as_list": true,
                "key": "attributes",
                "label": "Attributes",
                "use_label_wrap": true,
                "object_type": {
                    "type": "text"
                }
         }
}
```